pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -427,11 +485,13 @@ func TestUpdateStatus(t *testing.T) {
 }
 
 func newUserAccount(userName, userID string) *toolchainv1alpha1.UserAccount {
+	finalizers := []string{userAccFinalizerName}
 	userAcc := &toolchainv1alpha1.UserAccount{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      userName,
-			Namespace: "toolchain-member",
-			UID:       types.UID(uuid.NewV4().String()),
+			Name:       userName,
+			Namespace:  "toolchain-member",
+			UID:        types.UID(uuid.NewV4().String()),
+			Finalizers: finalizers,
```

Comment:
Don't add finalizers for every preexisting account. We must test the logic which adds the finalizer in our controller during reconciling.

Reply:
I need to look further into this. Without adding it here, many of the other tests fail